### PR TITLE
Amp list layout

### PR DESCRIPTION
**Pull request by NataliaLKB**
Trialling out the popular list in AMP to be more like story cards. 

image

In the next PR:
- add styling for different tones

cc @stephanfowler

### CODE REVIEW

File: common/app/views/fragments/amp/stylesheets/mostPopular.scala.html [hunk 1]
```diff
@@ -12,25 +12,33 @@
...
+    border-top: 0.0625rem solid #4bc6df;
+    padding-left: 8.166666875rem;
+    position: relative;
+    min-height: 84px;
+    margin-bottom: 10px;
+}
+
+.fc-item__image-container {
+    position: absolute;
+    margin-left: -7.916666875rem;
```
Comment (Jholder112233) [inline common/app/views/fragments/amp/stylesheets/mostPopular.scala.html]: -7.916666875rem :dizzy_face:?

Comment (NataliaLKB) [inline common/app/views/fragments/amp/stylesheets/mostPopular.scala.html]: Haha - copied from our styling on the site.

**Comment by NataliaLKB** [comment]
@Jholder112233 take another look?

**Comment by NataliaLKB** [comment]
@stephanfowler gave a verbal :+1: merging :)

File: common/app/views/fragments/amp/stylesheets/mostPopular.scala.html [hunk 2]
```diff
@@ -12,25 +12,33 @@
...
-    float: left;
-    width: 3.75rem;
-    color: #dcdcdc;
-    font-weight: 200;
-    font-size: 2.75rem;
-    line-height: 2.75rem;
+.fc-item {
+    background-color: #f6f6f6;
+    border-top: 0.0625rem solid #4bc6df;
+    padding-left: 8.166666875rem;
```
Comment (Jholder112233) [inline common/app/views/fragments/amp/stylesheets/mostPopular.scala.html]: Why so many dp?

**Pull request merged**